workflows/cloudify_system_workflows/snapshots/utils.py

```diff
@@ -34,20 +34,25 @@
 SCHEMA_SCRIPT = '/opt/manager/resources/cloudify/migrations/schema.py'
 
 
-class DictToAttributes(object):
-    def __init__(self, dic):
-        self._dict = dic
-
-    def __getattr__(self, name):
-        return self._dict[name]
+class DictToAttributes(dict):
+    def __init__(self, dictionary=None):
+        super(DictToAttributes, self).__init__()
+        if dictionary is None:
+            pass
+        elif isinstance(dictionary, dict):
+            for key in dictionary:
+                self.__setitem__(key, dictionary[key])
+        else:
+            raise TypeError('expected dict')
+
+    def __getattr__(self, key):
+        return super(DictToAttributes, self).__getitem__(key)
+
+    def __setattr__(self, key, value):
+        super(DictToAttributes, self).__setitem__(key, value)
 
     def __str__(self):
-        try:
```

Comment:
Same

Reply:
__str__ is used to print the config dictionary in the function '_print_postgres_config(config)' under postgres.py.
Otherwise it would print the dictionary keys and values as unicode (i.e. {u'key': u'value'}).
Should we drop it??